Zend/tests/lazy_objects/oss_fuzz_71407.phpt

```diff
@@ -0,0 +1,22 @@
+--TEST--
+oss-fuzz #71407
+--FILE--
+<?php
+
+class C {
+    public $ÿ=ÿ;
```

Comment:
Nit: Can we remove the special char? Doesn't seem like it's relevant to the test.